logger/test_log_file_merger.py

```diff
@@ -0,0 +1,84 @@
+"""
+Log file merger unit tests.
+"""
+
+import os
+import tempfile
```

Comment:
Remove. Pytest already provides a fixture that provides a path to a temporary directory that is deleted: `tmp_path: pathlib.Path` . You might need to create the directory when using it though: `tmp_path.mkdir(parents=True, exist_ok=True)`